lib/funky/page.rb

```diff
@@ -0,0 +1,43 @@
+module Funky
+  class Page < FacebookGraphRootNode
+    # For further detailed documentation, see
+    # https://developers.facebook.com/docs/graph-api/reference/page/
+
+    # @return [String] the name of the Facebook Page.
```

Comment:
@claudiofullscreen - updated... curious, why is the code review for the documentation style changing now? I was following the original style this was done in....